File: core/src/test/java/com/sequenceiq/cloudbreak/service/messages/TestConfig.java
package com.sequenceiq.cloudbreak.service.messages;

import org.springframework.boot.test.context.TestConfiguration;
import org.springframework.context.annotation.Bean;

import com.sequenceiq.cloudbreak.message.CloudbreakMessagesService;

@TestConfiguration
public class TestConfig {

    @Bean
    public CloudbreakMessagesService cloudbreakMessagesService() {
        return new CloudbreakMessagesService();
    }

}

File: core/src/test/java/com/sequenceiq/cloudbreak/service/messages/TestConfig.kt
package com.sequenceiq.cloudbreak.service.messages

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class TestConfig {

    @Bean
    fun cloudbreakMessagesService(): CloudbreakMessagesService {
        return CloudbreakMessagesService()
    }

}
